### DISCUSSION TITLE
Add setting for DecimalField's normalize_output

### DISCUSSION THREAD

**Discussion by savvan0h** [encode/django-rest-framework#8856] [Ideas & Suggestions] [answered]
I'd like to have a default setting for DecimalField's [normalize_output](https://github.com/encode/django-rest-framework/pull/6514), as suggested [here](https://github.com/encode/django-rest-framework/pull/6514#issuecomment-725713189).

```python
REST_FRAMEWORK = {
    ...
    'NORMALIZE_DECIMAL_OUTPUT': True,
}
```

Happy to open a PR for this.

**Comment by auvipy** [comment, marked as answer]
At this stage of the project, adding new API surface to the framework is net negative.
